git_machete/github.py

```diff
@@ -214,6 +214,18 @@ def __fire_github_api_request(method: str, path: str, token: Optional[str], requ
                 f'`{method} {url}` request ended up in 404 response from GitHub. A valid GitHub API token is required.\n'
                 f'Provide a GitHub API token with `repo` access via one of the: {get_github_token_possible_providers()} '
                 'Visit `https://github.com/settings/tokens` to generate a new one.')  # TODO (#164): make dedicated exception here
+        elif err.code == http.HTTPStatus.TEMPORARY_REDIRECT:
+            current_repo_and_org = get_repo_and_org_names_by_id(err.headers['Location'].split('/')[4])
+            new_path = f'/repos/{current_repo_and_org}/{"/".join(path.split("/")[4:])}'
```

Comment:
Hmmm how is the `new_path` gonna look like? 🤔 maybe add an example in comment in code

Reply:
And/or extract the `"/".join(path.split("/")[4:])` part to a separate variable